bbl_screen-patch/patches/printerui/qml/X1Plus.js

```diff
@@ -45,21 +46,33 @@ X1Plus.BedMeshCalibration = X1PlusBedMeshCalibration;
 var BedMeshCalibration = X1PlusBedMeshCalibration;
 X1Plus.ShaperCalibration = X1PlusShaperCalibration;
 var ShaperCalibration = X1PlusShaperCalibration;
+X1Plus.GpioKeys = X1PlusGpioKeys;
+var GpioKeys  = X1PlusGpioKeys;
 
 Stats.X1Plus = X1Plus;
 DDS.X1Plus = X1Plus;
 BedMeshCalibration.X1Plus = X1Plus;
 ShaperCalibration.X1Plus = X1Plus;
+GpioKeys.X1Plus = X1Plus;
 
 var _DdsListener = JSDdsListener.DdsListener;
 var _X1PlusNative = JSX1PlusNative.X1PlusNative;
-
 var DeviceManager = null;
 var PrintManager = null;
+var PrintTask = null;
```

Comment:
I realized that it might be possible to pull these out from the Binding shim?  I should try that some day rather than this Awaken nonsense to pass context around.